examples/step-3/doc/results.dox

```diff
@@ -225,7 +223,7 @@ suggestions:
 <h3>Using the %HDF5 Library to output the solution and additional data</h3>
 %HDF5 is a commonly used format that can be read by many scripting languages (e.g. R or Python). Here are some 
 insights on what is possible.
-<h4> Changing the output from .gnuplot to .h5:</h4>
+<h4> Changing the output to .h5:</h4>
```

Comment:
If you wouldn't mind, please remove this change again. It's going to conflict with my changes in #9250 where I make the same kind of change.

Reply:
Done. Sorry for the conflict.